src/index.html

```diff
@@ -1,12 +1,329 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <img
+        class="logo"
+        src="../src/images/logo.png"
+        alt="Moyo logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__items">
+            <a
+              class="nav__cell is-active"
+              href="#apple"
+              data-qa="card"
+            >Apple</a>
+            <div class="bottom-line"></div>
```

Comment:
Just the suggestion but you can make such line by using css pseudoelement :after.